src/libinjector/injector.c

```diff
@@ -116,7 +118,7 @@ injector_status_t injector_start_app(
     const char* binary_path,
     const char* target_process,
     bool break_loop_on_detection,
-    injector_t* injector_to_be_freed,
+    void** injector_to_be_freed,
```

Comment:
Switching this to `void**` I don't think is helping with readability of the code.

Reply:
I agree. But I don't know how to fix this properly. Client code does not know, `linux_injector_t` or `injector_t` will return. This hack needed only for windows use case now.

Reply:
Done